lib/util/query.js

```diff
@@ -0,0 +1,22 @@
+import coreUtils from '@opentripplanner/core-utils'
+import qs from 'qs'
+
+const { planParamsToQueryAsync } = coreUtils.query
+
+/** Extracts a query object from a query object, using an optional OTP configuration. */
+export async function getQueryParamsFromQueryObj (queryObj, config) {
+  // Filter out the OTP (i.e. non-UI) params.
+  const planParams = {}
+  Object.keys(queryObj).forEach(key => {
+    if (!key.startsWith('ui_')) planParams[key] = queryObj[key]
+  })
+
+  // Fully-format query object.
+  return planParamsToQueryAsync(planParams, config)
+}
+
+/** Extracts a query object from a query string, using an optional OTP configuration. */
+export async function getQueryParamsFromQueryString (queryString, config) {
+  const queryObj = qs.parse(queryString.split('?')[1])
+  return getQueryParamsFromQueryObj(queryObj, config)
+}
```

Comment:
Are these util functions used anywhere? If not can they be removed?

Reply:
Good catch. Removed in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/d77e617082a85151315a22e478befc5629fd8ba7.